index.bs

```diff
@@ -2976,8 +2978,8 @@ an assertion. Its {{PublicKeyCredentialRequestOptions/challenge}} member MUST be
         See {{UserVerificationRequirement}} for the description of {{AuthenticatorSelectionCriteria/userVerification}}'s values and semantics.
 
     :   <dfn>extensions</dfn>
-    ::  This OPTIONAL member contains [=client extension inputs=] requesting additional processing by the [=client=] and [=authenticator=].
-        The extensions framework is defined in [[#sctn-extensions]].
+    ::  The [=[RP]=] MAY use this OPTIONAL member to provide [=client extension inputs=]
+        requesting additional processing by the [=client=] and [=authenticator=].
```

Comment:
Nit:  this language is fine by me but I suggest not losing the cross-reference to [[#sctn-extensions]]:
```suggestion
    ::  The [=[RP]=] MAY use this OPTIONAL member to provide [=client extension inputs=]
        requesting additional processing by the [=client=] and [=authenticator=] (the extensions framework is defined in [[#sctn-extensions]]).
```

Reply:
Good point, thanks!